fvm/state/accounts.go

```diff
@@ -233,13 +233,17 @@ func contractKey(contractName string) string {
 }
 
 func (a *Accounts) getContract(contractName string, address flow.Address) ([]byte, error) {
+
+	fmt.Printf("TEMP LOGGING: get contract called for %s.%s \n", address.String(), contractName)
```

Comment:
This log and the one below should probably use a logger

Reply:
applied at 7a8f7f3, also reduced it to one log only.